code/drasil-example/template/lib/Drasil/Template/Body.hs

```diff
@@ -62,8 +71,8 @@ refDB :: ReferenceDB
 refDB = rdb [] []
 
 -- MOVE TO CONCEPTS
-example :: CI -- name of example
-example = commonIdeaWithDict "example" (pn "Template") "Template" []
+template :: CI
+template = commonIdeaWithDict "templateName" (pn "Template") "Template" []
```

Comment:
I think the point of having it named "example" is so that changing its internal name (i.e., "Template") wouldn't require changing the variable name itself throughout the code (i.e., `example`), but I guess Haskell's type checking/compile time errors would help us make this change (case in point: you were able to do it!) - I seem to recall other examples using `example`, `progName`, or something similar. I would be curious to see what @smiths thinks.

Reply:
What is the purpose of the concept `template`?  My instinct is that the previous `example` is a better concept to use, if the purpose of the template is to show the variation points that will be changed when the template is instantiated for a specific example.  We aren't currently looking at changing the template, so `templateName` seems odd to me.  I feel like our goal for the template is to show a generic template that highlights where a specific example will make changes.  The users will not be creating a new template, but rather a new example following the template.

However, I could very well be missing the point of the `template` CI.

Reply:
The reason I did it was that the title of the document changed from "Software Requirements Specification for Template" to "Software Requirements Specification for example". So in order to counter that I changed the way that it was defined.

Reply:
Can we change the title to "Software Requirements Specification for ProgName".  @samm82 is correct that I often use ProgName as the generic thing that is being referred to.

Reply:
Changed!